app/src/main/java/com/github/sdp_begreen/begreen/models/event/Event.kt

```diff
@@ -22,4 +22,29 @@ interface Event<T> : CopyableWithId<T>, Parcelable {
     val rootPath: RootPath
 
     override fun equals(other: Any?): Boolean
+
+    /**
```

Comment:
I am not sure why you would like to have a null boolean as a possibility of return here ? For example, if the starting date is null, for me the event did not start therefore it should be false. What do you think about it ? 

If you agree, you can make little modification like this : 

fun isStarted(): Boolean {
    return startDateTime?.let { it < System.currentTimeMillis() } ?: false
}

fun isFinished(): Boolean {
    return endDateTime?.let { it < System.currentTimeMillis() } ?: false
}

fun isActive(): Boolean {
    return isStarted() && !isFinished()
}

Reply:
This is what I wrote initially, but I think that if the initial date is null, then the event is not started neither not started, and we can know that from the return value of `isActive` (which will return null). With your implementation I think we lose some of the information, so if you don't mind I prefer to keep it like this. Do not hesitate to continue this conversation if you really think I'm wrong tho 😃

Reply:
I was also thinking the same but I thought it was not very relevant. However, since @bryanporcello also noticed it, I think it's better to change it. In general, it's weird to return an optional Bool